robot-agent/logMonitor.js

```diff
@@ -6,6 +6,8 @@ const { getLogger, topicToPath} = require('@transitive-sdk/utils');
 const log = getLogger('logMonitor.js');
 log.setLevel('info');
 
+const LOGS_UPLOADING_INTERVAL = 100; // Interval to process logs in milliseconds
```

Comment:
That is very aggressive. What happens when upload keeps failing, how much load would this put on the robot?